test/parallel/test-stream-manualstart.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
+
+fs.promises.open(__filename).then(common.mustCall((fd) => {
+  const rs = new fs.ReadStream(null, {
```

Comment:
This test `fs.ReadStream`, not `stream.Readable`. While this test is correct, can you please use one that just use `stream.Readable`?

Reply:
A `Readable` always starts in paused mode, this change affects only a `ReadStream`

Reply:
> A Readable always starts in paused mode

How is that relevant? It behaves the same as ReadStream. The same test should apply.

Reply:
> A `Readable` always starts in paused mode, this change affects only a `ReadStream`

If that was true, you'd not need to modify `Readable`.

Reply:
Maybe there is some biggus problem, how do you think?

Reply:
> Maybe there is some biggus problem, how do you think?

Likely, but it's possible to add an independent test.

Reply:
I'm confused what the problem here is? We just need change ReadStream to Readable. What is the "biggus" problem?